### DISCUSSION TITLE
Feature request: Default band is last band worked or 20m

### DISCUSSION THREAD

**Discussion by TrystanDavies** [magicbug/Cloudlog#844] [Ideas / Feature Requests]
Hi,

I hadn't realised the forum had gone, but this is just as good. I hope I'm posting this in the correct place.

The default band when cloudlog is started up is 160m, and I'm guessing most people don't have a great deal of QSOs there as the efective antennas are rather huge. 

Would it be at all possible to either set the default band in a settings option, or set the default band to the last band that was logged?

It's not a massively important request, but it would be good if when I press "Live QSO" it went straight to a band I was likely to be on.

Thanks,
Trystan (Paypal contributor to the project)
G0KAY

**Comment by magicbug** [comment]
Pleased you found it, hopefully via redirect, makes sense to keep it all in one place.

custom mode stuff is on the todo.

**Reply by TrystanDavies** [reply]
Good man. Yes, I found it via redirect, so all is well.

Trystan
G0KAY

**Comment by TrystanDavies** [comment]
Hi, 

I really would like this request, please, to default "Live QSO" to be the last band that was used when last logged in, rather than "160m".

I placed this request over a year ago, so I hope I can't be accused of being impatient.

Many thanks,
Trystan G0KAY

**Comment by c-o-m-m-a-n-d-e-r-d-a-t-a** [comment]
+1 for this please... Cloudlog is outstanding, and the responsive mobile interface is a boon when operating mobile or portable, but having to fiddle around with the band and mode for software that's otherwise so versatile seems like a glaring oversight in the user experience!

**Reply by int2001** [reply]
We created the "Favorites" (upper right "star" at live-qso) for that. 
You're absolutely right that the 160m is very annoying. I will have a look at setting the qrg/mode/etc persistent to the last used one.

Maybe the new user_option-table could be a solution for that.

**Reply by c-o-m-m-a-n-d-e-r-d-a-t-a** [reply]
Amazing, thanks for the response! Fav band / mode is a help, to be sure. The less buttons to press, the better, especially when you're out portable in the howling wind and rain. If I may humbly feature-request also an obvious shortcut button to a new live QSO at the top of the front page (at least on the responsive mobile interface?) A bit like the material-design paradigm of apps where we see a single button for the most often used task. Thanks a million. Cloudlog rocks.

**Reply by int2001** [reply]
Have you tried CloudLogOffline? If you're in the field, this is a huge improvement instead of fiddling around with the website. You can also upload the QSOs direct to Cloudlog (as bulk). Give it a chance:

Android: https://play.google.com/store/apps/details?id=de.webappjung.CloudLogOffline&hl=gsw&gl=US
iOS: https://apps.apple.com/de/app/cloudlogoffline/id1528219213

The Quicklog-Thing at the Frontpage has been discussed several times. Mainproblems are here:
- usability (bloating up cloudlog)
- Where to put QRG/Mode/etc.
i don't think that we will see a Quicklog-Button so fast. Sorry

**Comment by c-o-m-m-a-n-d-e-r-d-a-t-a** [comment]
I didn't know of it, but I have installed it and it looks great, although
uploading via the API key doesn't seem to work for me although I have tried
configuring it. Also compiled and tested on desktop, same deal. I wonder if
it's because my cloudlog site is running in a virtual folder ie /cloudlog
on my server, and the config asks for an actual hostname with separate http
/ https rather than a URL, suggesting it might not handle a virtual
folder... I will raise an issue
in https://github.com/myzinsky/cloudLogOffline/issues . Adif export appears
to work OK but kind of removes the advantage of an integrated app - I'd
rather use the web interface, it's pretty outstanding regardless of a
Quicklog-Thing! Thanks again!

-----Original Message-----
From: Joerg
Reply-To: magicbug/Cloudlog
To: magicbug/Cloudlog
Cc: c-o-m-m-a-n-d-e-r-d-a-t-a, Comment
Subject: Re: [magicbug/Cloudlog] Feature request: Default band is last band
worked or 20m (Discussion #844)
Date: 15/10/23 11:05:54

Have you tried CloudLogOffline? If you're in the field, this is a huge
improvement instead of fiddling around with the website. You can also
upload the QSOs direct to Cloudlog (as bulk). Give it a chance:

Android:
https://play.google.com/store/apps/details?id=de.webappjung.CloudLogOffline&hl=gsw&gl=US
iOS: https://apps.apple.com/de/app/cloudlogoffline/id1528219213

The Quicklog-Thing at the Frontpage has been discussed several times.
Mainproblems are here:
- usability (bloating up cloudlog)
- Where to put QRG/Mode/etc.
i don't think that we will see a Quicklog-Button so fast. Sorry

**Comment by phl0** [comment]
What about re-using what we have?

![Screenshot from 2023-10-15 13-57-13](https://github.com/magicbug/Cloudlog/assets/7112907/62897f03-b0e0-4e67-9649-7d51e1c405aa)

However I would rename it so that the title is more general like "Default band used in QSO logging and gridmap" or similar. Opinions @int2001 @AndreasK79 @magicbug ?

**Reply by int2001** [reply]
Good idea. But what should we do in case of „SAT“?

**Reply by phl0** [reply]
Ignore I would say....

**Reply by magicbug** [reply]
Yep think i agree with Flo on this one

**Comment by phl0** [comment]
Solution proposed in https://github.com/magicbug/Cloudlog/pull/2566. Feedback welcome.
